### DISCUSSION TITLE
Syncrhonize once in a while

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [samschott/maestral#694] [Ideas]
I use [this software](https://github.com/abraunegg/onedrive) as OneDrive client.

Here there is no daemon running all the time if you don't want to.

You just call `onedrive --synchronize` and it will synchronize your local folder with the remote one.

Is this something possible with maestral?

**Comment by samschott** [comment]
That's not currently possible, its either continuous syncing or none at all. You can however wait for a sync cycle to be completed and pause syncing / stop maestral again :)

I can see the functionality being useful if file changes are primarily made only on one side (either server or locally).
